python/pycarbon/integration/tensorflow.py

```diff
@@ -0,0 +1,358 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""A set of Tensorflow specific helper functions for the unischema"""
```

Comment:
```suggestion
"""A set of Tensorflow specific helper functions"""
```

Reply:
Is there any example code to use tensorflow to read carbon to train a model?

Reply:
done.  Mnist: tf_external_example_carbon_unified_api.py